release/cloudbuild.yaml

```diff
@@ -36,11 +36,13 @@ steps:
       - output
   # Copy the artifacts from the Cloud Build VM to GCS.
   - name: gcr.io/cloud-builders/gcloud
+    entrypoint: bash
     args:
+      - gcloud
```

Comment:
I'd like @jiggoha to take a look at this entrypoint swapping business.

Reply:
I think the reason Al has added that here is so that the `sha256sum` and other bash commands work correctly. The environment variables here get populated correctly not because of bash environment variables, but because of Cloud Build substitution vars. So without the entrypoint, we can't do any logic here. Is that right, Al?

Reply:
Yup, that's right